static/style/search.css

```diff
@@ -167,7 +167,7 @@ form.compact {
     }
   }
 
-  &.has-attention {
+  &[data-expanded="true"] {
```

Comment:
This is like a simple rename "data-expanded" just has the standard collapsed/expanded notion.  I literally already forgot what "has-attention" actually means, and I think it is exactly that.